lib/truffle/java/java_utilities.rb

```diff
@@ -224,6 +224,7 @@ def self.make_proxy(a_class)
             Java.invoke_java_method(CLASS_GET_SIMPLE_NAME, a_class)),
           a_proxy) unless parent == nil
       rescue
+        nil
```

Comment:
@aardvark179 could you have a look at some of these rescues in java interoperability files. It should be possible for you to push to this branch (PR). I'll deal with the rest.